velox/vector/tests/utils/VectorTestBase.h

```diff
@@ -15,15 +15,16 @@
  */
 #pragma once
 
+#include <gtest/gtest.h>
+#include <optional>
+
 #include <folly/Executor.h>
 #include <folly/executors/CPUThreadPoolExecutor.h>
 
+#include "velox/exec/MemoryReclaimer.h"
```

Comment:
Is it feasible to use utilities from velox::exec within velox::vector? It may cause cycle dependencies?